WalletWasabi/Extensions/NBitcoinExtensions.cs

```diff
@@ -482,13 +482,11 @@ public static int EstimateInputVsize(this Script script)
 
 		public static T FromBytes<T>(byte[] input) where T : IBitcoinSerializable, new()
 		{
-			BitcoinStream inputStream = new BitcoinStream(input);
+			BitcoinStream inputStream = new(input);
 			var instance = new T();
 			inputStream.ReadWrite(instance);
-			if (!inputStream.IsEnd())
-				throw new FormatException("Expected end of stream");
 
-			return instance;
+			return !inputStream.IsEnd() ? throw new FormatException("Expected end of stream") : instance;
 		}
 
 		public static bool IsEnd(this BitcoinStream bitcoinStream)
```

Comment:
I copied this extension method without much thinking. It could be removed or used as an local function. I mean, `BitcoinStream` is used in hundreds of places in NBitcoin and this new method was never necessary so, clearly we can remove it.

Reply:
@lontivero Please check if what I did is correct https://github.com/zkSNACKs/WalletWasabi/pull/5312/commits/fb3b54f35429f1380b1cc581cfbd036e49f7e670